app/Http/Controllers/OrganizationController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Resources\OrganizationResource;
+use App\Models\Organisation;
+
+class OrganizationController extends Controller
+{
+    public function __invoke()
+    {
+        return OrganizationResource::collection(
+            Organisation::query()
```

Comment:
```suggestion
            Organisation::query()
                ->withoutEagerLoads(['city'])
                ->select(['id', 'name', 'county_id'])
```

- `->withoutEagerLoads(['city'])` ca să excludem eager load-ul înregistrat în model.
- `->select(['id', 'name', 'county_id'])` ca să nu tragem din db mai multe date decât necesar.